### PR TITLE
Add basic android project

### PR DESCRIPTION
**Pull request by Velaron**
This is heavily unfinished, but is in a fully working state. Would be good to have this merged until I find inspiration to work on the Android project further...

### CODE REVIEW

**Comment by nekonomicon** [comment]
> android/app/src/main/java/in/celest/xash3d/cs16client/MainActivity.java

cs16client?

**Comment by Velaron** [comment]
> > android/app/src/main/java/in/celest/xash3d/cs16client/MainActivity.java
> 
> cs16client?

oops, fixed

File: android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    packagingOptions {
+        jniLibs {
+            useLegacyPackaging = true
+        }
+    }
+
+    defaultConfig {
+        applicationId 'com.example.hlsdk'
+        versionName '1.35'
```
Comment (nekonomicon) [inline android/app/build.gradle:19]: 1.35?

**Comment by a1batross** [comment]
Looks good enough to me.

**Review by a1batross — APPROVED** [review]

**Pull request merged**